packages/sources/tradermade/src/endpoint/forex.ts

```diff
@@ -0,0 +1,9 @@
+import { ExecuteWithConfig, Config } from '@chainlink/types'
+import * as live from './live'
+
+export const supportedEndpoints = ['forex']
+
+export const execute: ExecuteWithConfig<Config> = async (input, context, config) => {
```

Comment:
This is a bit weird but it looks like we only want to limit WS support for `forex` pairs.

Reply:
I'm not sure if I'm missing something, but couldn't we add `forex` in the `supportedEndpoints` array? And shouldn't they be switched around?

Not sure I fully understand when do we want HTTP only vs when do we want to enable WS requests? Isn't it enough with only one endpoint and controlling WS with `WS_ENABLED`?

Reply:
that's what I had thought a well but it sounded like Bryan wanted equities to keep using http and for only forex to use WS.  I'll confirm with him again during standup.

Reply:
Confirmed with Bryan that we can collapse this into a single endpoint.